src/prefect/engine.py

```diff
@@ -214,6 +213,7 @@ async def create_then_begin_flow_run(
     engine_logger.info(f"Created flow run {flow_run.name!r} for flow {flow.name!r}")
 
     if state.is_failed():
+        flow_run_logger(flow_run).error(state.message)
```

Comment:
I think this log chronologically makes more sense up on line 200 with `.exception` instead of `.error`

Reply:
I think it's in that order to avoid the strange situation of logging in the order:
>> Flow run <flow_run_name> received invalid parameters and is marked as failed
>> Created flow run <flow_run_name> for flow <flow.name>

Which I think could be confusing. Definitely updating to exception.